<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_Calibration</class>
 <widget class="QWidget" name="widget_Calibration">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>833</width>
     <height>377</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>24</y>
      <width>153</width>
      <height>273</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>模块型号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>检测类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>硬件检测：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>错误警告：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>24</y>
      <width>321</width>
      <height>273</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_Module_Type">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>LX - XXXX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Object_Type">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Hard_Detection_Code">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>FFFF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Error">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>无错误</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar_Calibration">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>320</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1a1a1a;
</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
